ms2query/query_from_sqlite_database.py

```diff
@@ -7,13 +7,17 @@
 from typing import Dict, List
 import numpy as np
 import pandas as pd
+from tqdm import tqdm
 import sqlite3
 from matchms.Spectrum import Spectrum
 
 
 def get_spectra_from_sqlite(sqlite_file_name: str,
                             spectrum_id_list: List[str],
-                            table_name: str = "spectrum_data"
+                            spectrum_id_storage_name: str = "spectrumid",
```

Comment:
`spectrum_id_storage_name` sounded a bit cryptic. But I also don't have a great suggestions. `spectrum_id_field`?

Reply:
I think the best solution is actually making a class for sqlite database, so instead of having to pass this everytime to all query_from_sqlite_functions it is just an attribute of the sqlite database class (and a class would make sense for more things) Then the name might still be a bit cryptic, but at least it does not show up in that many unrelated functions. I made an issue for this #69